pimd/pim_cmd.c

```diff
@@ -5665,12 +5672,43 @@ static void show_multicast_interfaces(struct pim_instance *pim, struct vty *vty)
 		}
 
 		ifaddr = pim_ifp->primary_address;
+		if (uj) {
+			json_row = json_object_new_object();
+			json_object_string_add(json_row, "name", ifp->name);
+			json_object_string_add(json_row, "state",
+					       if_is_up(ifp) ? "up" : "down");
+			json_object_string_add(
+				json_row, "address",
+				inet_ntoa(pim_ifp->primary_address));
+			json_object_int_add(json_row, "ifIndex", ifp->ifindex);
+			json_object_int_add(json_row, "vif",
+					    pim_ifp->mroute_vif_index);
+			json_object_int_add(json_row, "packetsIn",
```

Comment:
What I see the problem here that we should use CLI/JSON names identically where possible. We had the same problem in BGP and it's eventually fixed somewhere. We should use here `pktsIn` and so on.

Reply:
@ton31337 Done